internal/devbox/shell.go

```diff
@@ -347,12 +349,13 @@ func (s *DevboxShell) writeDevboxShellrc() (path string, err error) {
 	return path, nil
 }
 
-// linkShellStartupFiles will link files used by the shell for initialization.
-// We choose to link instead of copy so that changes made outside can be reflected
-// within the devbox shell.
```

Comment:
sigh, this comment was obsolete as of two years ago, after: https://github.com/jetify-com/devbox/pull/1587/files